include/boost/histogram/detail/probit.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+#define BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+
+#include <cmath>
+
+# define PI 3.14159265358979323846  /* pi */
+
+namespace boost {
+namespace histogram {
+namespace detail {
+
+template <class T>
+T ErfInv(T x){
+   T tt1, tt2, lnx, sgn;
+   sgn = (x < 0) ? -1.0f : 1.0f;
```

Comment:
In C++, it is best to use std::copysign, https://en.cppreference.com/w/cpp/numeric/math/copysign